src/components/Error.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components/macro";
+
+function Message() {
```

Comment:
Be more precise with the naming. Message sounds like a generic message component. How about NoActivitiesMessage

Reply:
Thx, changed it.